core/src/test/java/com/sequenceiq/distrox/v1/distrox/converter/InstanceTemplateParameterConverterTest.java

```diff
@@ -198,6 +198,15 @@ void convertTestAwsInstanceTemplateV1ParametersToAwsInstanceTemplateV4Parameters
 
     }
 
+    @Test
+    void testWhenDatahubEncryptionTypeSetToNull() {
+        AwsInstanceTemplateV1Parameters source = new AwsInstanceTemplateV1Parameters();
+        source.setEncryption(encryption(null, null));
+        DetailedEnvironmentResponse environment = createDetailedEnvironmentResponseForAwsEncryption(true, false, null);
+        AwsInstanceTemplateV4Parameters awsInstanceTemplateV4Parameters = underTest.convert(source, environment);
+        assertThat(awsInstanceTemplateV4Parameters).isNotNull();
+    }
```

Comment:
Please also assert that:
* `awsInstanceTemplateV4Parameters.getEncryption()` is not `null`.
* `awsInstanceTemplateV4Parameters.getEncryption().getKey()` and `awsInstanceTemplateV4Parameters.getEncryption().getType()` are both `null`.

Reply:
Addressed this.